reqs/requirements-dev-lite.txt

```diff
@@ -1,12 +1,10 @@
 boto3==1.17.106 # This should match the version in constraints-dev.txt
-darglint>=1.8.1
 flask>=1.0.0 # for s3 test only (with moto)
 freezegun>=0.3.15
 ipykernel<=6.17.1 # Newest version (6.19.0 released on 12/7/2022) causes "WARNING  traitlets:client.py:1181 No handler found for comm target 'comm'" by "https://github.com/jupyter/nbclient/blob/main/nbclient/client.py" (version 0.7.2) to be emitted.
 mock-alchemy>=0.2.5
 moto>=2.0.0,<3.0.0
 nbconvert>=5
-pydocstyle>=6.1.1
```

Comment:
curious: does `ruff` not need to be included in the `requirements` file because we include it in `pyproject.toml`?

Reply:
`ruff` is already included in `requirements-dev-contrib.txt` thanks to a recent PR from @Kilo59 :)

Reply:
(Which is installed in the docstring_linter CI stage via `reqs/requirements-dev-test.txt`)